### PR TITLE
Deps array update.

### PR DESCRIPTION
**Pull request by nicolad**
## Problem this Pull Request solves
While I was migrating code to barista, the eslint from that project notified me about some deps that needs to be updated.
Seems that ESLint isn't working properly here.

### CODE REVIEW

**Comment by nicolad** [comment]
> Did you already make these changes in barista or just in the core repo?
if the latter then maybe we should hold off on merging until the migration is finished?

yes, I made these changes also in barista

File: assets/src/domain/eventEditor/ui/tickets/ticketPriceCalculator/components/DefaultPricesInfo.tsx [hunk 1]
```diff
@@ -1,26 +1,33 @@
...
-				{__('Modify default prices.')}
-			</a>
-		</div>
+		hasDefaultPrice && (
+			<div className='ee-tpc__default-prices-info'>
+				<a href={href} target='_blank' rel='noopener noreferrer' onClick={onClickLink}>
+					{__('Modify default prices.')}
+				</a>
+			</div>
+		)
```
Comment (nicolad) [inline assets/src/domain/eventEditor/ui/tickets/ticketPriceCalculator/components/DefaultPricesInfo.tsx:30]: these changes have been updated from barista

File: assets/src/application/ui/input/Button/Link.tsx [hunk 2]
```diff
@@ -15,7 +15,7 @@ const Link: React.FC<LinkProps> = ({ children, href, icon, tooltip, tooltipProps
 	);
 
 	const link = (
-		<a href={href} className={className} target='_blank' rel='noopener norefferer'>
+		<a href={href} className={className} target='_blank' rel='noopener noreferrer'>
```
Comment (nicolad) [inline assets/src/application/ui/input/Button/Link.tsx:18]: https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types/noreferrer

Comment (tn3rb) [reply assets/src/application/ui/input/Button/Link.tsx:18]: oh wow good catch

File: assets/src/application/ui/display/icons/HelpIcon/index.tsx [hunk 3]
```diff
@@ -8,24 +8,18 @@ import { IconProps } from '../types';
...
 interface HelpIconProps extends IconProps {
-    clickable?: boolean;
-    tooltipText?: string;
+	clickable?: boolean;
+	tooltipText?: string;
 }
 
 const HelpIcon: React.FC<HelpIconProps> = ({ clickable, tooltipText, ...props }) => {
-    const className = classNames('ee-help-icon', props.className);
+	const className = classNames('ee-help-icon', props.className);
```
Comment (manzoorwanijk) [inline assets/src/application/ui/display/icons/HelpIcon/index.tsx:17]: Is indentations messed up here?

Comment (nicolad) [reply assets/src/application/ui/display/icons/HelpIcon/index.tsx:17]: yes, I had some issues with VS Code few months ago and ended up reinstalling it fully

**Pull request merged**